sycl/include/CL/sycl/detail/clusm.hpp

```diff
@@ -1,100 +0,0 @@
-//==---------------- clusm.hpp - SYCL USM for CL Utils ---------*- C++ -*---==//
-//
-// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
-// See https://llvm.org/LICENSE.txt for license information.
-// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
-//
-// ===--------------------------------------------------------------------=== //
-
-#pragma once
-
-#include <CL/cl.h>
-#include <CL/cl_usm_ext.h>
-#include <CL/sycl/detail/os_util.hpp>
-#include <CL/sycl/detail/pi.hpp>
-
-#include <map>
-#include <mutex>
-#include <vector>
-
-__SYCL_INLINE_NAMESPACE(cl) {
-namespace sycl {
-namespace detail {
-namespace usm {
-
-class CLUSM {
-public:
-  CLUSM() = default;
-  ~CLUSM() = default;
-
-  void *hostMemAlloc(cl_context context, cl_mem_properties_intel *properties,
```

Comment:
Did these methods have implementation?
We should remove it's file too, I believe.

Reply:
I believe that was previously done.